### PR TITLE
Disable node creation with right mouse when brushing

### LINKED ISSUE #4364: Right click in hybrid brush-mode also creates node
When dragging the mouse while right-clicking, everything works fine. However, not moving the mouse while clicking will brush AND create a skeleton node which is wrong.

### PR DESCRIPTION
**Pull request by MichaelBuessemeyer**
This PR disabled node creation with right-click while in brushing mode in hybrid tracings.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a hybrid tracing. Select the brush mode and try to create nodes using right-click. 
- Do keep the button pressed as this does not trigger a click action.

### Issues:
- fixes #4364

------
- [x] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)
- ~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~
- ~~[ ] Updated [documentation](../blob/master/docs) if applicable~~
- ~~[ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes~~
- ~~[ ] Needs datastore update after deployment~~
- [x] Ready for review

### CODE REVIEW

**Comment by MichaelBuessemeyer** [comment]
@philippotto I also added #4427. I am not sure which bounding box to disable, the user bb or the task bb :confused:

**Comment by philippotto** [comment]
> @philippotto I also added #4427. I am not sure which bounding box to disable, the user bb or the task bb

I think, both should be disabled :)

File: frontend/javascripts/oxalis/controller/scene_controller.js [hunk 1]
```diff
@@ -251,6 +251,11 @@ class SceneController {
       showCrossSections: true,
     });
     this.userBoundingBox.getMeshes().forEach(mesh => this.rootNode.add(mesh));
+    const { viewMode } = Store.getState().temporaryConfiguration;
+    if (viewMode === "flight") {
+      // We disable the bounding box in flight mode as it distracts the user.
+      this.userBoundingBox.setVisibility(false);
```
Comment (philippotto) [inline frontend/javascripts/oxalis/controller/scene_controller.js]: Does this need to be undone when switching to ortho mode? Or is the scene controller re-initialized in that case, anyway?

Comment (MichaelBuessemeyer) [reply frontend/javascripts/oxalis/controller/scene_controller.js]: * When ortho mode is loaded `startPlaneMode` is called. There all bounding boxes are set visible again.
* When the user changes to any arbitrary mode the `stopPlaneMode`-method is called that makes all bounding boxes invisible.

I hope this answers your question :D

**Comment by MichaelBuessemeyer** [comment]
@philippotto Could you please review these changes again?

File: frontend/javascripts/oxalis/controller/scene_controller.js (hunk 1 above)
Comment (philippotto) [reply frontend/javascripts/oxalis/controller/scene_controller.js]: Hm, then I don't understand why it's necessary to set the visibility to false here, at all? At the end of `createMeshes` `this.stopPlaneMode();` is called which should hide all boxes, anyway?

Comment (MichaelBuessemeyer) [reply frontend/javascripts/oxalis/controller/scene_controller.js]: Ok, your are completely correct here. I missed that :see_no_evil: . Thanks for noticing

Comment (MichaelBuessemeyer) [reply frontend/javascripts/oxalis/controller/scene_controller.js]: But in `buildTaskingBoundingBox` we still need to eventually disable the task bounding box, as this method is also called each time the task bounding box is changed in the store.

Or are you confident that the task bounding box will never change in the store?

Comment (philippotto) [reply frontend/javascripts/oxalis/controller/scene_controller.js]: > But in buildTaskingBoundingBox we still need to eventually disable the task bounding box, as this method is also called each time the task bounding box is changed in the store.

Yes, you are right, the code which reacts to store changes should set the correct visibility :+1:

**Comment by MichaelBuessemeyer** [comment]
@philippotto Could you please re-review this PR?

File: frontend/javascripts/oxalis/controller/scene_controller.js [hunk 2]
```diff
@@ -289,12 +289,16 @@ class SceneController {
...
       }
 
+      const { viewMode } = Store.getState().temporaryConfiguration;
       this.taskBoundingBox = new Cube({
         min: taskBoundingBox.min,
         max: taskBoundingBox.max,
         color: 0x00ff00,
         showCrossSections: true,
       });
+      if (viewMode === "flight") {
```
[the remark this answers is not in the document]
Comment (MichaelBuessemeyer) [reply frontend/javascripts/oxalis/controller/scene_controller.js]: I change this to 
```
if (constants.MODES_ARBITRARY.includes(viewMode)) 
```
That should be fine, right?

[the remark this answers is not in the document]
Comment (philippotto) [reply frontend/javascripts/oxalis/controller/scene_controller.js]: Yes, even better :+1:

**Pull request merged**